lib/fluent/winsvc.rb

```diff
@@ -63,7 +63,7 @@ def service_main
         loop do
           sleep 5
           break unless running?
-          raise Errno::ECHILD unless Process.waitpid2(@pid, Process::WNOHANG)
+          raise Errno::ECHILD if Process.waitpid2(@pid, Process::WNOHANG)
```

Comment:
```suggestion
          raise Errno::ECHILD if Process.waitpid(@pid, Process::WNOHANG)
```
We don't use the second return value in actual.

Reply:
Thanks! Fixed!